pkg/render/guardian.go

```diff
@@ -95,14 +95,26 @@ func (c *GuardianComponent) service() *corev1.Service {
 			Selector: map[string]string{
 				"k8s-app": GuardianName,
 			},
-			Ports: []corev1.ServicePort{{
-				Port: 9200,
-				TargetPort: intstr.IntOrString{
-					Type:   intstr.Int,
-					IntVal: 8080,
+			Ports: []corev1.ServicePort{
+				{
+					Name: "elasticsearch",
+					Port: 9200,
+					TargetPort: intstr.IntOrString{
+						Type:   intstr.Int,
+						IntVal: 8080,
+					},
+					Protocol: corev1.ProtocolTCP,
 				},
-				Protocol: corev1.ProtocolTCP,
-			}},
+				{
+					Name: "kibana",
+					Port: 5601,
+					TargetPort: intstr.IntOrString{
+						Type:   intstr.Int,
+						IntVal: 8080,
```

Comment:
Is this value correct? I'm not sure the implementation, but it seems strange that we have two different ports resolving to the same target port.

Also, does the target port need to be different from the service port?

Reply:
Yes to both questions. Applications will send requests to tigera-secure-kb-http.tigera-kibana.svc:5601 and tigera-secure-es-http.tigera-elasticsearch.svc:9200, so we have to open both 9200 and 5601. The tunnel excepts requests on 8080, so that has to be the target port.